scripts/build-image.sh

```diff
@@ -24,7 +24,13 @@ function usage {
   echo -e "Environment variable:"
   echo -e " PUSH_TO_REGISTRY     \tPush images to caicloud registry or not, options: Y or N. Default value: ${PUSH_TO_REGISTRY}"
 }
-
+function disable-selinux() {
+  if selinuxenabled && [ "$(getenforce)" = "Enforcing" ]; then
```

Comment:
I run your PR in OS X 10.12.1, and it raises an error:

```
./scripts/build-image.sh: line 28: selinuxenabled: command not found
```

maybe we should define it before this function :)